Project.toml

```diff
@@ -14,19 +14,19 @@ Symbolics = "0c5d862f-8b57-4792-8d23-62f2024744c7"
 Aqua = "0.8"
 CSVFiles = "1"
 Catalyst = "13"
-DataFrames = "1"
-DataStructures = "0.18"
-DiffEqBase = "6"
-LinearAlgebra = "1"
-ModelingToolkit = "8.47"
-OrdinaryDiffEq = "6"
-Plots = "1"
-SparseArrays = "1"
-Sundials = "4"
-Symbolics = "5.0.3"
+DataFrames = "1.3"
+DataStructures = "0.18.13"
+DiffEqBase = "6.117"
+LinearAlgebra = "1.10"
+ModelingToolkit = "8.51"
+OrdinaryDiffEq = "6.47"
+Plots = "1.10"
+SparseArrays = "1.10"
+Sundials = "4.4"
+Symbolics = "5.2"
 TimerOutputs = "0.5"
 Test = "1"
-julia = "1.6"
+julia = "1.10"
```

Comment:
@ChrisRackauckas is SciML now moving to Julia 1.10 and greater going forward or should this be a lower version cap?

Reply:
Yes. Note that there's big discussions of v1.10 being the new LTS. It's not official yet, but we needed to bump a few things (many libraries could no longer support pre-1.9) so since it's 95% sure v1.10 is the new LTS, we're just going to bump it all to v1.10 and hope and pray. If we need to backport any fixes we can but this should drastically simplify things.

Reply:
OK, thanks for letting me know. I'll keep this in mind going forward across libraries.